class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
```

Comment:
```suggestion
	 * Check to see if a block is able to run with current WordPress or Gutenberg plugin versions
```

Reply:
Expanded the comment here to make it clearer what was being checked.